src/java.desktop/share/classes/java/awt/image/BufferedImage.java

```diff
@@ -1540,6 +1540,13 @@ public void setData(Raster r) {
   /**
    * Adds a tile observer.  If the observer is already present,
    * it receives multiple notifications.
+   *
+   * @implSpec
```

Comment:
I don't think this needs to be implSpec. I think we can say : 
This method ignores its parameters and does nothing, since {@code BufferedImage} is always checked out for writing and cannot be made read-only, so there can never be events to dispatch.